### DISCUSSION TITLE
Even if I set the size to 0, I can still see a white vertical line in the middle of the screen. How should I remove it

### DISCUSSION THREAD

**Discussion by LionelXN** [unoplatform/uno#11696] [Q&A]
![image](https://user-images.githubusercontent.com/18021306/225252522-870e5804-ecac-4965-a087-7ef39bec47a6.png)
![image](https://user-images.githubusercontent.com/18021306/225252655-d9b2c162-08a3-497d-b437-a0f6fb5358f7.png)

**Comment by LionelXN** [comment]
And when I set the size to 200 and start the app, two forms will be displayed in the upper left corner and middle of the screen
![image](https://user-images.githubusercontent.com/18021306/225253786-84602377-3ea7-4366-b1ea-c3d7a3aca2a8.png)
![image](https://user-images.githubusercontent.com/18021306/225253843-e1a2c8b9-9e43-4766-a400-a2480e0affcf.png)

**Reply by jeromelaban** [reply]
Thanks for the report. What the actual desired intent? What are you trying to acheive?

**Reply by LionelXN** [reply]
Thanks for your reply，i want to remove or override the default window style, including the form border and title bar, just like WPF,is there any way?

**Reply by jeromelaban** [reply]
Thanks for the update. The GtkHost class exposes its window:

https://github.com/unoplatform/uno/blob/6a14547498d791c334fb47c8997acbe4e7cab948/src/Uno.UI.Runtime.Skia.Gtk/GtkHost.cs#L83

And you should be able to interact with this API to change its visual aspect.
